README.md

```diff
@@ -54,16 +55,27 @@ In this step, we'll view the same HSSD scene on the Quest VR headset. No HITL se
+
+Assets for the Habitat client aren't included in the Unity project (aside from a test asset in `Assets/temp/`). They must be imported separately. These are built using a special pipeline defined [here](https://github.com/facebookresearch/habitat-lab/blob/main/examples/siro_sandbox/VR_HITL.md#dataset-processing).
 
-1. Undo any local changes to `ScriptSingletonsObject` in the Inspector Pane. In particular, make sure `ReplayFileLoader` is disabled and `NetworkClient` is enabled.
-    * Remember, you can always do `git status` in the project root folder to check if you've accidentally made unwanted local changes (caveat: unsaved changes probably won't show up here, so remember to save often).
-3. Under `Network Client`, enter the server address and port where you'll run the HITL server. (This is probably the same machine where you're running the Unity Editor.)
-4. Follow instructions for [Testing the HITL server and Unity client together](https://docs.google.com/document/d/1cvKuXXE2cKchi-C_O7GGVFZ5x0QU7J9gHTIETzpVKJU/edit#bookmark=id.l3y1pdpos6t).
-5. See also [Playing Fetch in the VR client](https://docs.google.com/document/d/1cvKuXXE2cKchi-C_O7GGVFZ5x0QU7J9gHTIETzpVKJU/edit#bookmark=id.uqh8z8qcn7y8) for tips on what to do in the scene.
+## Troubleshooting
+### Deployment to the VR Headset
 
-# Troubleshooting
-* Beware you may see a runtime error in the Unity Console about a null reference inside XR Device Input code. I haven't tracked this down, but it appears to be a bug with the XR Device Simulator and it appears to be harmless. However, it should be fixed soon because it is causing console spam and potentially masking important errors.
-* You may see some errors about missing GLB models. In some cases, this is expected, e.g. we've intentionally omitted a high-poly model. In other cases, you might have set up your data incorrectly. In the Project pane, make sure you have e.g. `Assets/Resources/data/fpss/stages/106879104_174887235.glb`.
-* If you have problems getting the client to connect to the server, one thing to test is whether a minimal client can connect to the websocket connection. This minimal client can be a simple python app or simple HTML/Javascript page (the latter can be served from the server via a HTTP server and loaded in the Quest browser). Ask Eric U for instructions.
+* Make sure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is activated.
+* When connecting your headset to your development computer via USB, a pop-up will ask you to confirm the connection within the VR headset.
+  * If the pop-up doesn't show up, reconnect your USB cable.
+  * You may also have to re-do this after the headset goes into sleep mode.
+* Deployment occasionally fails when the application is already installed. You can delete the old build from the Quest storage menu. The following error will often show in Unity when that occurs:
```

Comment:
"You can delete the old build from the Quest storage menu (find it under 'unknown sources')"